packages/@secretlint/quick-start/src/index.ts

```diff
@@ -1,8 +1,10 @@
+import { fileURLToPath } from "node:url";
+import path from "node:path";
 import { cli, run } from "secretlint/cli";
 
 export const quickStart = async () => {
     // Override secretlintrc with quick-start pkg if it is not defined.
-    const defaultSecretlintRc = new URL("../config/.secretlintrc.json", import.meta.url).pathname;
+    const defaultSecretlintRc = path.resolve(fileURLToPath(import.meta.url), "../config/.secretlintrc.json");
```

Comment:
fileURLToPath(import.meta.url) point to packages/@secretlint/quick-start/module/index.js

```
> path.resolve("packages/@secretlint/quick-start/src/index.ts", "../config/.secretlintrc.json")
'/Users/azu/ghq/github.com/textlint/textlint/packages/textlint/packages/@secretlint/quick-start/src/config/.secretlintrc.json'
```

```suggestion
    const __dirname = path.dirname(fileURLToPath(import.meta.url));
    const defaultSecretlintRc = path.resolve(__dirname, "../config/.secretlintrc.json");
```

is correct?

Reply:
Thanks for your help! I agree with it.